lib/sql_footprint/sql_anonymizer.rb

```diff
@@ -3,6 +3,7 @@ class SqlAnonymizer
     GSUBS = {
       /\sIN\s\((.*)\)/ => ' IN (values-redacted)'.freeze, # IN clauses
       /\s\=\s([0-9]+)/ => ' = number-redacted'.freeze, # numbers
+      /\s\!\=\s([0-9]+)/ => ' != number-redacted'.freeze, # numbers w/ not equals
```

Comment:
would something like `/\s.\=\s([0-9]+)/` give us what @schneiderderek is talking about? 
or `/\s[!<>]{0,1}\=\s([0-9]+)/` 
Then maybe you could capture the thing to put it in the number-redacted string?